queries/__init__.py

```diff
@@ -253,6 +253,546 @@ def convert(m):
+        "hk": {"nf": "fyrst", "þf": "fyrst", "þgf": "fyrstu", "ef": "fyrsts"},
+    },
+    "ft": {
+        "kk": {"nf": "fyrstir", "þf": "fyrsta", "þgf": "fyrstum", "ef": "fyrstra"},
+        "kvk": {"nf": "fyrstrar", "þf": "fyrstar", "þgf": "fyrstum", "ef": "fyrstra"},
+        "hk": {"nf": "fyrst", "þf": "fyrst", "þgf": "fyrstum", "ef": "fyrstra"},
+    },
+}
+
+_SUB_20_NEUT_TO_ORDINAL = {
+    "eitt": "fyrst",
+    # 2 is a special case
+    "þrjú": "þriðj",
+    "fjögur": "fjórð",
+    "fimm": "fimmt",
+    "sex": "sjött",
+    "sjö": "sjöund",
+    "átta": "áttund",
+    "níu": "níund",
+    "tíu": "tíund",
+    "ellefu": "elleft",
+    "tólf": "tólft",
+    "þrettán": "þrettánd",
+    "fjórtán": "fjórtánd",
+    "fimmtán": "fimmtánd",
+    "sextán": "sextánd",
+    "sautján": "sautjánd",
+    "átján": "átjánd",
+    "nítján": "nítjánd",
+}
+
+_ANNAR_TABLE = {
+    "et": {
+        "kk": {
+            "nf": "annar",
+            "þf": "annan",
+            "þgf": "öðrum",
+            "ef": "annars",
+        },
+        "kvk": {
+            "nf": "önnur",
+            "þf": "aðra",
+            "þgf": "annarri",
+            "ef": "annarrar",
+        },
+        "hk": {
+            "nf": "annað",
+            "þf": "annað",
+            "þgf": "öðru",
+            "ef": "annars",
+        },
+    },
+    "ft": {
+        "kk": {
+            "nf": "aðrir",
+            "þf": "aðra",
+            "þgf": "öðrum",
+            "ef": "annarra",
+        },
+        "kvk": {
+            "nf": "aðrar",
+            "þf": "aðrar",
+            "þgf": "öðrum",
+            "ef": "annarra",
+        },
+        "hk": {
+            "nf": "önnur",
+            "þf": "önnur",
+            "þgf": "öðrum",
+            "ef": "annarra",
+        },
+    },
+}
+
+_SUB_20_ORDINAL_SUFFIX = {
+    "kk": {
+        "nf": "i",
+        "þf": "a",
+        "þgf": "a",
+        "ef": "a",
+    },
+    "kvk": {
+        "nf": "a",
+        "þf": "u",
+        "þgf": "u",
+        "ef": "u",
+    },
+    "hk": {
+        "nf": "a",
+        "þf": "a",
+        "þgf": "a",
+        "ef": "a",
+    },
+}
+
+_TENS_NEUT_TO_ORDINAL = {
+    "tuttugu": "tuttug",
+    "þrjátíu": "þrítug",
+    "fjörutíu": "fertug",
+    "fimmtíu": "fimmtug",
+    "sextíu": "sextug",
+    "sjötíu": "sjötug",
+    "áttatíu": "átttug",
+    "níutíu": "nítug",
+}
+
+_LARGE_ORDINAL_SUFFIX = {
+    "kk": {
+        "nf": "asti",
+        "þf": "asta",
+        "þgf": "asta",
+        "ef": "asta",
+    },
+    "kvk": {
+        "nf": "asta",
+        "þf": "ustu",
+        "þgf": "ustu",
+        "ef": "ustu",
+    },
+    "hk": {
+        "nf": "asta",
+        "þf": "asta",
+        "þgf": "asta",
+        "ef": "asta",
+    },
+}
+
+
+def _num_to_ordinal(
+    word: str, decl: str = "nf", gender: str = "kk", number: str = "et"
+) -> str:
+    """
+    Helper function. Changes one part of a number (in written form) to ordinal form
+    in correct declension, gender and number.
```

Comment:
In most cases (pun not intended) we use "case" for "fall".

Reply:
Yes, that is the case (pun intended) ;).

Reply:
Fixed!